_infra/helm/locust/locustfiles/locustfile.py

```diff
@@ -478,19 +478,31 @@ def post(self, url: str, data: dict = {}):
 class FrontstageTasks(TaskSet, Mixins):
 
     def on_start(self):
-        self.sign_in()
```

Comment:
We don't really want to remove this from on_start, the idea is that the respondent logs in once and then clicks around the site, therefore making it more like a real journey

Reply:
I have restored the sign_in() function to what it was.